Sources/hostmgr/commands/vm/VMStart.swift

```diff
@@ -74,17 +74,18 @@ struct VMStartCommand: AsyncParsableCommand {
 
     var sharedPaths: [LaunchConfiguration.SharedPath] {
         get throws {
-            guard try FileManager.default.directoryExists(at: Paths.gitMirrorStorageDirectory) else {
-                return []
+            var paths: [LaunchConfiguration.SharedPath] = []
+
+            if try FileManager.default.directoryExists(at: Paths.gitMirrorStorageDirectory)
+                && self.withGitMirrors == true {
```

Comment:
Nitpick: If you test for `self.withGitMirrors` _first_, that would allow you to prevent an unnecessary filesystem I/O operation for nothing when `withGitMirrors` is `false` 🙃 

```suggestion
            if self.withGitMirrors && try FileManager.default.directoryExists(at: Paths.gitMirrorStorageDirectory) {
```

Reply:
This resulted in `'try' cannot appear to the right of a non-assignment operator`.
I've added the argument for using common credentials and fixed it on 2e1260c39b46702ca45db4848ab8e5968f0da76b.

Reply:
👍
A bit disappointed that we can't test both conditions in that order in a single condition and need to use nested ifs…  wonder f we're missing some nice Swift syntax that would allow us that…